ldap-connector/processes/setupConfig.mod

```diff
@@ -0,0 +1,63 @@
+[Ivy]
+17D5C4CA5704F5EC 9.3.1 #module
+>Proto >Proto Collection #zClass
+sg0 setupConfig Big #zClass
+sg0 B #cInfo
+sg0 #process
+sg0 @AnnotationInP-0n ai ai #zField
+sg0 @TextInP .type .type #zField
+sg0 @TextInP .processKind .processKind #zField
+sg0 @TextInP .xml .xml #zField
+sg0 @TextInP .responsibility .responsibility #zField
+sg0 @StartSub f0 '' #zField
+sg0 @EndSub f1 '' #zField
+sg0 @GridStep f3 '' #zField
+sg0 @PushWFArc f4 '' #zField
+sg0 @PushWFArc f2 '' #zField
+>Proto sg0 sg0 setupConfig #zField
+sg0 f0 inParamDecl '<> param;' #txt
+sg0 f0 outParamDecl '<com.axonivy.connector.ldap.util.JndiConfig jndiConfig> result;' #txt
+sg0 f0 outParamTable 'result.jndiConfig=in.jndiConfig;
+' #txt
+sg0 f0 callSignature call() #txt
+sg0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>call()</name>
+    </language>
+</elementInfo>
+' #txt
+sg0 f0 81 49 30 30 -13 17 #rect
+sg0 f1 337 49 30 30 0 15 #rect
+sg0 f3 actionTable 'out=in;
+' #txt
+sg0 f3 actionCode 'import ch.ivyteam.naming.JndiProvider;
+import com.axonivy.connector.ldap.util.JndiConfig;
+
+in.jndiConfig = JndiConfig.create()
```

Comment:
could be possible that more settings should be configurable over variables.
at least the connection timeout.

Reply:
done. but how to setup a demo without providing user and pw?

Reply:
I think you need to make a Demo with an HTML Dialog with two more textboxes for username and password.